src/lib/containers/AccessRequestSubmissionTable.tsx

```diff
@@ -0,0 +1,191 @@
+    } else {
+      setSort({ field, direction: 'DESC' })
+    }
+  }
+
+  return (
+    <div className="bootstrap-4-backport">
+      <div className="SRC-split">
+        {accessorId ? (
+          <Typography variant="headline3" style={{ marginBottom: 0 }}>
+            Submissions including <UserOrTeamBadge principalId={accessorId} />
+          </Typography>
+        ) : (
+          <Typography variant="headline3" style={{ marginBottom: 0 }}>
+            Access Request Submissions
+          </Typography>
+        )}
+      </div>
+
+      <div className="AccessSubmissionTable">
+        <Table striped borderless bordered={false}>
+          <thead>
+            <tr>
+              <th>REQUEST</th>
+              <th>Access Requirement Name</th>
+              {showSubmitter && <th>Submitter</th>}
+              {showStatus && <th>Status</th>}
+              {showRequestors && <th>Requestors</th>}
+              <th>Reviewer(s)</th>
+              <th>
+                Created Date
+                <SortIcon
+                  role="button"
+                  style={{ float: 'right' }}
+                  active={sort.field === 'CREATED_ON'}
+                  aria-label="Sort by Created On"
+                  direction={
+                    sort.field === 'CREATED_ON'
+                      ? sort.direction === 'DESC'
+                        ? 'DESC'
+                        : 'ASC'
+                      : 'DESC'
+                  }
+                  onClick={() => onSort(SubmissionSortField.CREATED_ON)}
+                />
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {accessSubmissions.map(item => {
+              return (
+                <tr key={item.id}>
+                  <td>
+                    <a
+                      href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!AccessRequirement:AR_ID=${item.id}`}
+                    >
+                      {item.id}
+                    </a>
+                  </td>
+                  <td>{item.accessRequirementName}</td>
+                  {showSubmitter && (
+                    <td>
+                      <UserOrTeamBadge principalId={item.submitterId} />
+                    </td>
+                  )}
+                  {showStatus && (
+                    <td>{upperFirst(item.state.toLocaleLowerCase())}</td>
+                  )}
+                  {showRequestors && (
+                    <td>
+                      <UserOrTeamBadge principalId={item.submitterId} />
+                      {item.accessorChanges
+                        .filter(user => item.submitterId !== user.userId)
+                        .map(requestor => (
+                          <UserOrTeamBadge
+                            key={requestor.userId}
+                            principalId={requestor.userId}
+                          />
+                        ))}
+                    </td>
```

Comment:
I think the design indicated a different style for non-submitters (I think non-submitters are italicized and lighter color, not sure)